fido2/hid/linux.py

```diff
@@ -56,13 +56,19 @@ def get_descriptor(path):
 
         # Read product
         buf = array("B", [0] * 128)
-        fcntl.ioctl(f, HIDIOCGRAWNAME, buf, True)
-        name = bytearray(buf).decode("utf-8") or None
+        ret = fcntl.ioctl(f, HIDIOCGRAWNAME, buf, True)
+        if ret > 1:
+            name = bytearray(buf[: (ret - 1)]).decode("utf-8") or None
```

Comment:
We can drop the `or None` here and on line 69 since we have the `ret > 1` check, right?

Reply:
Yes, I think so. The `decode()` would throw an exception on bad UTF-8, anyway.